tests/x11/evolution/evolution_timezone_setup.pm

```diff
@@ -42,8 +42,8 @@ sub run {
     assert_and_click "mercator-projection";
     assert_and_click("mercator-zoomed-in-clicked");
     # Change timezone to Shanghai
-    if (check_screen "timezone-asia", 30) {
-        send_key_until_needlematch("timezone-asia", "down");
+    if (check_screen("timezone-asia")) {
+        send_key("right");
```

Comment:
I would recommend to squash commits. There is no need to track typos/fixes during test development and introduce new commits in history.